package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "chrome-dgram",
   "description": "Use the Node `dgram` API in Chrome Apps",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
It's best to keep the version change out of PRs so that can be done right before publishing time. Otherwise, if another release happens before this PR gets merged, then the version will be messed up.

No worries this time.